proto/internalapi/scanner/v4/index_report.proto

```diff
@@ -0,0 +1,22 @@
+// This contains protobuf types in pair with ClairCore's types. See
+// https://github.com/quay/claircore for comments on the fields.
+
+syntax = "proto3";
+
+package v4;
```

Comment:
nit: I'm thinking scannerv4 because I think we can potentially have issues if the rest of StackRox updates to v4. We've had issues when both Scanner and StackRox used v1 in the past.

Reply:
I can't fully recall everything, but I found this PR in the scanner repo which addressed the concerns (doesn't explain anything, though): https://github.com/stackrox/scanner/pull/266

From what I can recall, we had a naming collision in the proto namespace when we added some more protos in Scanner to be used by Central. I think there was a naming conflict because the package names were the same, even though they are in two different repos. I can't really remember all details, but I did find this which may explain the issue a bit: https://protobuf.dev/reference/go/faq/#namespace-conflict

I think the typical recommendation is to make the package name something that can minimize these conflicts, so I figure we should use `scannerv4`, so we can name anything we want whatever we want, and we won't have to worry about the rest of StackRox potentially having issues were they to eventually bump to a `v4`

Reply:
dab68010c6